composer.json

```diff
@@ -26,10 +29,14 @@
     },
     "require": {
         "php": ">=7.1",
+        "yiisoft/yii2": "~2.0.13",
```

Comment:
is it something special about 2.0.13 version that it needs to be added here? isn't yii2-httpclient enough?

Reply:
no, there is nothig special about 2.0.13, but yii2-httpclient is not enough any more if one is implementing `validator` which is part of `yiisoft/yii2`